### PR TITLE
Fix getItem() to stop returning empty string value as null

### PR DESCRIPTION
**Pull request by plainweaver**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
+.idea
+.DS_Store
+
```
Comment (gr2m) [inline .gitignore]: could you remove these unrelated changes? 

I recommend to create `.gitignore` folder in your user’s home directory, usually `~/.gitignore` where you can add these lines

**Comment by gr2m** [comment]
The PR looks good otherwise, I’ll update the `.travis.yml` file to removed the outdated Node versions :)

**Comment by plainweaver** [comment]
on my way.
